index.html

```diff
@@ -16,26 +15,163 @@
+            <li class="category flex cross-center ellipse line-height highlight uppe">Back End Dev</li>
+            <li class="category flex cross-center ellipse line-height highlight uppe">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="tags flex">
+            <li class="line-height round-borders">html</li>
+            <li class="line-height round-borders">css</li>
+            <li class="line-height round-borders">javaScript</li>
+          </ul>
+          <button class="bg round-borders cap" type="button">see project</button>
+        </div>
+      </article>
+
+      <article class="card flex flex-column bg">
+        <img src="images/snapshoot/snapshoot_4.svg" alt="project photo">
+        <div class="content">
+          <h3 class="heading bold cap">Multi-Post
+            Stories</h3>
+          <ul class="info flex">
+            <li class="category flex cross-center ellipse line-height highlight uppe">CANOPY</li>
+            <li class="category flex cross-center ellipse line-height highlight uppe">Back End Dev</li>
+            <li class="category flex cross-center ellipse line-height highlight uppe">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="tags flex">
+            <li class="line-height round-borders">html</li>
+            <li class="line-height round-borders">css</li>
+            <li class="line-height round-borders">javaScript</li>
+          </ul>
+          <button class="bg round-borders cap" type="button">see project</button>
+        </div>
+      </article>
+    </div>
+  </section>
+
+  <section class="aboutMe margin-seprator bg">
+   <div class="wrapper hero flex space-between flex-column">
+  <div class="left-side hero flex space-between flex-column">
+    <h2 class="heading">About
+      Myself</h2>
+    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work
+      and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <a href="#">Let's contact</a>
+    <ul class="social">
+      <li><a href="#"><img src="images/social/twitter.svg" alt="twitter"></a></li>
+      <li><a href="#"><img src="images/social/linkedin.svg" alt="linkedin"></a></li>
+      <li><a href="#"><img src="images/social/medium.svg" alt="medium"></a></li>
+      <li><a href="#"><img src="images/social/github.svg" alt="github"></a></li>
+      <li><a href="#"><img src="images/social/vector.svg" alt="vector"></a></li>
+    </ul>
+    <button class="bg round-borders cap" type="button">Get my resume</button>
+  </div>
+
+  <details class="ability" open>
+    <summary>
+      <span> Langguages</span>
```

Comment:
- [ ] As this is a subtitle of the section, kindly use an hx tag like h3 instead of a span, this is very important for accessibility when a user is using a screen reader, as well, as defining the start of sub-sections in HTML
- [Optional] Also kindly fix the typo with "Languages", it's marked as optional, because the template design has the typo, but it'd probably be better to still fix it, to make your portfolio look more professional